Sources/Build/ManifestBuilder.swift

```diff
@@ -510,6 +510,8 @@ extension LLBuildManifestBuilder {
             if target.underlyingTarget is SystemLibraryTarget { return }
             // Ignore Binary Modules.
             if target.underlyingTarget is BinaryTarget { return }
+            // Ignore Extension Targets.
+            if target.underlyingTarget is ExtensionTarget { return }
```

Comment:
The reason we don't reflect the extension target's dependencies here is because we will later set up build rules that establish that establishes that dependency (just below).